MVCCore.Models/Doctor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Runtime;
+
+namespace MVCCore
+{
+    public class Doctor
+    {
+        [Key]
+        public int DoctorId { get; set; }
+        [Required]
+        public string DoctorName { get; set; }
+        [Required]
+        public string DoctorAddress { get; set; }
+        [Required]
+        public int DoctorAge { get; set; }
```

Comment:
Remove data validation attribute when you using fluent validation, this fires before the fluent in the .net pipeline

Reply:
Removed